datafusion/sqllogictest/test_files/information_schema.slt

```diff
@@ -202,7 +202,7 @@ datafusion.execution.parquet.pruning true
 datafusion.execution.parquet.pushdown_filters false
 datafusion.execution.parquet.reorder_filters false
 datafusion.execution.parquet.skip_metadata true
-datafusion.execution.parquet.statistics_enabled NULL
+datafusion.execution.parquet.statistics_enabled page
```

Comment:
I think this is an improvement -- it doesn't change the default value (`NULL` means use arrow-rs defaults, which is `page`), but now the default value isexplicit in the config settings

Also there is a test to ensure the defaults don't drift from the arrow-rs defaults accidentally